pkgs/top-level/perl-packages.nix

```diff
@@ -19684,10 +19684,10 @@ let
 
   SysVirt = buildPerlModule rec {
     pname = "Sys-Virt";
-    version = "7.0.0";
+    version = "7.5.0";
```

Comment:
Does it need to be bumped to 7.6.0?

Reply:
I don't know if it is *needed* but i know its *wanted*.

Reply:
It does not seem like SysVirt will be updated to 7.6.0 any time soon, it has been about 2 weeks and I have seen no sign of it 7.6.0 being released, so I decided to stick to 7.5.0.

Reply:
Sys-Virt is now at 7.7.0